<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginDockWidgetBase</class>
 <widget class="QDockWidget" name="PluginDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="allowedAreas">
   <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>MLA Paddock Power</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <spacer name="toolbarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="currentTimeframeButton">
        <property name="toolTip">
         <string>Show the Property as it is now</string>
        </property>
        <property name="text">
         <string>Current</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/plugins/mlapp/images/current.png</normaloff>:/plugins/mlapp/images/current.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="futureTimeframeButton">
        <property name="toolTip">
         <string>Show the Property with all planned changes</string>
        </property>
        <property name="text">
         <string>Future</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/plugins/mlapp/images/future.png</normaloff>:/plugins/mlapp/images/future.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="sketchFenceButton">
        <property name="toolTip">
         <string>Sketch a Fence …</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/plugins/mlapp/images/new-fence.png</normaloff>:/plugins/mlapp/images/new-fence.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="sketchPipelineButton">
        <property name="toolTip">
         <string>Sketch a Pipeline …</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/plugins/mlapp/images/new-pipeline.png</normaloff>:/plugins/mlapp/images/new-pipeline.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="sketchWaterpointButton">
        <property name="toolTip">
         <string>Sketch a Waterpoint …</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/plugins/mlapp/images/new-waterpoint.png</normaloff>:/plugins/mlapp/images/new-waterpoint.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
